website/docs/guides/importing_resources.html.markdown

```diff
@@ -0,0 +1,304 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: Importing resources"
+sidebar_current: "docs-vcd-guides-importing-resources"
+description: |-
+ Provides guidance to VMware Cloud resources importing
+---
+
+# Importing resources
+
+Supported in provider *v3.11+* with Terraform *v1.5.x+* (tested with Terraform v1.5.4).
```

Comment:
If we're declaring 1.5.x support, we should remove the internal kitchen detail on which version we have tested. It is too much information and also may change any day:

```suggestion
Supported in provider *v3.11+* with Terraform *v1.5.x+*.
```

Reply:
fixed